product-service/src/services/response.js

```diff
@@ -0,0 +1,30 @@
+const response = (data) => {
+  return {
```

Comment:
here and below, if function returns the result only, you can omit 'return' keyword and just return the object